newsroom/company_admin/views.py

```diff
@@ -26,15 +26,27 @@ def get_view_data():
     company = get_company(user) or {}
     company_users = list(query_resource("users", lookup={"company": ObjectId(company["_id"])}))
     products = get_products_by_company(company)
+    sections = get_translated_sections(app.sections)
 
     return {
         "users": company_users,
         "companyId": str(company["_id"]),
         "companies": [company],
-        "sections": app.sections,
+        "sections": sections,
         "products": products,
     }
 
+def get_translated_sections(sections):
+    translated_sections = []
+    for section in sections:
+        translated_section = {
+            "_id": section["_id"],
+            "name": gettext(section["name"]),  # Translate the section name
```

Comment:
we should rather call the initial registraion, eg:
```
app.section("agenda", "Agenda", "agenda")
```
with `lazy_gettext` around `Agenda`:
```
app.section("agenda", lazy_gettext("Agenda"), "agenda")
```